docs/how-to/set-base-image.md

```diff
@@ -0,0 +1,14 @@
+# How to set base image
+
+This charm supports deploying the runners on different base images.
```

Comment:
Minor: The default is jammy, right? Should be described here in this docs or is already described somewhere else?

Reply:
That sounds like a good addition - the only place it mentions right now is in the `config.yaml`, thank you! (I'll push the change tomorrow morning - the runners seem to be very busy atm)